include/forcing/DataProvider.hpp

```diff
@@ -40,18 +40,18 @@ namespace data_access
 
         /** Return the variables that are accessable by this data provider */
```

Comment:
I just noticed a typo here
```suggestion
        /** Return the variables that are accessible by this data provider */
```
Probably not suitable to fix in this PR.